reference/spender.yaml

```diff
@@ -7810,6 +7810,88 @@ components:
               - id: Date
                 is_visible: true
                 order: 1
+    fastlink_params:
+      type: object
+      nullable: true
+      additionalProperties: false
+      properties:
+        configName:
+          type: string
+          maxLength: 255
+          description: |
+            Name of the configuration for the action that need to be performed, Aggregation in this case.
```

Comment:
Name of the configuration for the action that need to be performed, Aggregation in this case. -> `Name of the configuration for the action that would be performed while opening the Yodlee Fastlink application`